### DISCUSSION TITLE
Show Neogit rebase view when I run `git rebase -i`

### DISCUSSION THREAD

**Discussion by a-b** [NeogitOrg/neogit#1524] [Q&A]
I'm looking for community help with best practices for showing the Neogit rebase edit view every time I open the rebase tasks list `.git/rebase-merge/git-rebase-todo` by running `git rebase -i ..`

I'm using NvChad with Lazy.

**Comment by CKolkey** [comment]
Like so:
1. `:Neogit` Opens the status buffer
2. `r` Opens the rebase popup
3. `i` picks the "interactive" action
4. Choose a commit with <cr>
5. Enjoy the edit view :)

_or_
1. `:Neogit`
2. `l` Opens the log popup
3. `l`, `h`, `u`, `o` launch different projections of the graph
4. Find the commit you want to rebase onto
5. `r` then `i`

**Reply by NobbZ** [reply]
This implies starting the rebase from neogit.

OP is asking how to make `git rebase -i` open neovim in the correct mode.
